### PR TITLE
[RET-131] Vertically center page hero content

### PR DESCRIPTION
**Pull request by jcmagsay**
### What's this PR do?
Vertically center page hero content and remove some unnecessary code

### What are the relevant tickets?
https://mondorobot.atlassian.net/browse/RET-131

### Who should review this PR?
@jcmagsay @mhfen

### CODE REVIEW

File: src/assets/styles/modules/objects/heroes/page-hero.scss [hunk 1]
```diff
@@ -17,9 +17,6 @@
 }
 
 .pageHero-description {
-  align-items: center;
-  display: flex;
-  flex-direction: column;
+  @extend .container_centered;
```
Comment (mhfen) [inline src/assets/styles/modules/objects/heroes/page-hero.scss]: I wonder if you can accomplish the same thing by doing this:

```
<div class="pageHero">
  <div class="pageHero-description">
    <div class="container container_centered">
    </div>
  </div>
</div>
```
which would be more in line with our standards.

Comment (jcmagsay) [inline src/assets/styles/modules/objects/heroes/page-hero.scss]: Word, good call!

File: src/assets/styles/modules/objects/heroes/page-hero.scss [hunk 2]
```diff
@@ -3,12 +3,12 @@
 ------------------------- */
 
 .pageHero {
-  display: flex;
+  @extend .container_centered;
```
Comment (mhfen) [inline src/assets/styles/modules/objects/heroes/page-hero.scss]: Do we will need this with container being inside description now? Do we need `pageHero-description` at all?

Comment (jcmagsay) [inline src/assets/styles/modules/objects/heroes/page-hero.scss]: This has been resolved.

**Pull request merged**